src/require-strict-assert.ts

```diff
@@ -0,0 +1,70 @@
+// require-strict-assert.ts
+
+/*
+ * Copyright (c) 2021-2023 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import type { Rule } from 'eslint';
+
+export default {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Require importing strict version of node:assert and using non-strict assert functions.',
+      url: 'https://github.com/checkdigit/eslint-plugin',
+    },
+    fixable: 'code',
+  },
+  create(context) {
+    return {
+      Literal(node) {
+        if (node.value === 'node:assert' && node.parent.type === 'ImportDeclaration') {
+          const importDeclaration = node.parent;
+          const defaultSpecifier = importDeclaration.specifiers.find(
+            (specifier) => specifier.type === 'ImportDefaultSpecifier' || specifier.type === 'ImportNamespaceSpecifier',
+          );
+
+          if (defaultSpecifier) {
+            const importDeclarationRange = importDeclaration.range ?? [0, 0];
+            const rangeToReplace = defaultSpecifier.range ?? importDeclarationRange;
+            const correctedText = `{ strict as ${defaultSpecifier.local.name} }`;
+
+            context.report({
+              node: importDeclaration,
+              message: 'Require the strict version of node:assert.',
```

Comment:
"Require strict assertion mode" would be more compatible with the NodeJS assert terminology.